scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 673,
-  MAX_SIZE: 857,
+  MAX_SIZE: 875,
```

Comment:
The web-vitals package is 15kb minified, 50% of that is cross-fetch. we should looking into making this package smaller.